saleor/graphql/order/bulk_mutations/order_bulk_create.py

```diff
@@ -410,6 +434,9 @@ class OrderBulkCreateInput(graphene.InputObjectType):
     transactions = NonNullList(
         TransactionCreateInput, description="Transactions related to the order."
     )
+    invoices = NonNullList(
+        OrderBulkCreateInvoiceInput, description="Invoices related to the order."
+    )
```

Comment:
Maybe you are going to do that later in a separate PR, but just as a reminder: Keep in mind that the introduced types should get label `ADDED_IN_XX`. The `doc_category` should be also included in all introduced types.

Reply:
I will fix it on feature branch